### ISSUE TITLE
Docs not available on netlify - not found error

### ISSUE THREAD

**Issue by devandnow** [JustFly1984/react-google-maps-api#3282]

**Comment by Diego-Dev-C** [comment]
I think that it's been a while since the docs aren't available, crazy that this hasn't been fixed

**Comment by tomkoom** [comment]
up!

**Comment by ahollister** [comment]
Still appear to be down for me, is the URL still https://react-google-maps-api-docs.netlify.app/ ?

**Comment by tomkoom** [comment]
> Still appear to be down for me, is the URL still https://react-google-maps-api-docs.netlify.app/ ?

yes they are down, you can find more information on Slack, also you can access docs archive: https://web.archive.org/web/20230701010714/https://react-google-maps-api-docs.netlify.app/, but don't input your google api key on that page

**Comment by cristianTOTH** [comment]
this is sad :/

**Comment by ngochoangdev-bunbu** [comment]
The same error.
